src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -219,7 +219,7 @@ private static void EnsureHtmlParser()
 
             if (s_inputFieldRegex == null)
             {
-                s_inputFieldRegex = new Regex(@"<input\s+[^>]*(/>|>.*?</input>)",
+                s_inputFieldRegex = new Regex(@"<input\s+[^>]*(/>|>|>.*?</input>)",
```

Comment:
```suggestion
                s_inputFieldRegex = new Regex(@"<input\s+[^>]*(/?>|>.*?</input>)",
```

I think this is a bit of an easier to read / simpler regex expression that should have the same result.